matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -261,6 +309,28 @@ impl WebRtcSocket {
             .build()
     }
 
+    /// Gets a reference to the [`WebRtcChannel`] of a given id. May return [`None`] if
+    /// the channel has been taken.
+    ///
+    /// See also: [`WebRtcSocket::take_channel`]
+    pub fn channel(&mut self, channel: usize) -> Option<&mut WebRtcChannel> {
+        self.channels
+            .get_mut(channel)
+            .expect(&format!("No channel exists with id {channel}"))
```

Comment:
Document the panic or return None

Reply:
The panic is the same as the behaviour in the old `send_on_channel`, agree that a result would be nicer